setup/security/authorization/index.md

```diff
@@ -29,7 +29,7 @@ Permissions can be attached to applications and (provider) accounts. A permissio
 
 Keep these in mind as you consider your authorization strategy:
 
-1) Fiat's authorization model is a whitelist that is open by default. In other words, when a resource does _not_ 
+1) Fiat's authorization model open by default. In other words, when a resource does _not_
```

Comment:
In this case, I did not replace with another term as it seems like the point of this paragraph is that it is open by default, and didn't want to detract from that.